streams/src/main/java/org/apache/kafka/streams/kstream/internals/KTableImpl.java

```diff
@@ -220,6 +223,47 @@ public String queryableStoreName() {
         return doMapValues(mapper, new MaterializedInternal<>(materialized, builder, MAPVALUES_NAME));
     }
 
+    @Override
+    public <VR> KTable<K, VR> transformValues(ValueTransformerWithKeySupplier<? super K, ? super V, ? extends VR> transformerSupplier, String... stateStoreNames) {
+        return doTransformValues(transformerSupplier, null, stateStoreNames);
```

Comment:
Should we check `transformerSupplier` not null as well? Ditto below.

Reply:
Edit: I saw you've checked it in KTableTransformValues, which should be fine as well.